applications_tests/lethe-fluid-matrix-free/cylinder_kelly_estimator.prm

```diff
@@ -0,0 +1,186 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 2
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method            = steady
+  set number mesh adapt = 1
+  set output name       = output
+  set output frequency  = 0
+end
+
+#---------------------------------------------------
+# TESTING
+#---------------------------------------------------
+
+subsection test
+  set enable = true
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+
+subsection FEM
+  set velocity order = 1
+  set pressure order = 1
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+
+subsection timer
+  set type = none # <none|iteration|end>
+end
+
+#---------------------------------------------------
+# Force
+#---------------------------------------------------
+
+subsection forces
+  set verbosity             = verbose
+  set calculate force       = true
+  set calculate torque      = false
+  set output precision      = 10
+  set calculation frequency = 1
+  set output frequency      = 1
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+
+subsection initial conditions
+  set type = nodal
+  subsection uvwp
+    set Function expression = 1; 0; 0
+  end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection physical properties
+  set number of fluids = 1
+  subsection fluid 0
+    set kinematic viscosity = 1.0000
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type      = gmsh
+  set file name = ../cylinder_structured.msh
```

Comment:
Maybe:
```suggestion
  set file name = ../../cylinder_structured.msh
```

If this is not working, one needs to look at in which working directly we are and list all files there. I would say we are in `lethe/build/applications_tests/lethe-fluid-matrix-free/cylinder_kelly_estimator.debug/mpirun=2`.

Reply:
Yes, I will try to solve this now

Reply:
I have copied now the mesh also into this folder to ensure that we can use the same path in the parameter file for all the tests... let's see if it works now